src/layout/layout.rs

```diff
@@ -434,10 +440,27 @@ impl Layout {
         };
         let area_size = area_end - area_start;
 
+        let spacers_added = if layout.spacing == 0 {
+            false
+        } else {
+            matches!(
+                layout.flex,
+                Flex::Stretch | Flex::StretchLast | Flex::Start | Flex::Center | Flex::End
+            )
+        };
+        let constraints = if spacers_added {
+            Itertools::intersperse(
+                layout.constraints.iter().cloned(),
+                Constraint::Fixed(layout.spacing),
+            )
+            .collect_vec()
+        } else {
+            layout.constraints.iter().cloned().collect_vec()
```

Comment:
`.clone()` seems to be enough here
```suggestion
            layout.constraints.clone()
```